### ISSUE TITLE
Will this project have future developments?

### ISSUE THREAD

**Issue by imzyxwvu** [katlogic/lv8#1]

**Comment by katlogic** [comment]
It's actively used and fully working. All what is lacking is documentation. This is intentional - there will be probably api changes, especially wrt the 'this' -> 'self' mapping.

For example usage, you can refer to https://github.com/katlogic/nodemu/blob/master/init.lua (though that project is abandoned in favor of JS->Lua transpiler).

**Comment by imzyxwvu** [comment]
I used this library for lua5.2->JS calling on Windows in the past. It caused an application exception in JS for-in a Lua object. I forgot how to cause this bug. But I thought this project can be improved.

**Comment by imzyxwvu** [comment]
And this library depends on the platform very much. I suggest to remove the IO bindings. If I want to provide JS IO APIs, I would pass the _G.io to JS.

**Comment by katlogic** [comment]
Yes, its entirely linux specific, surprised it worked in windows at all.
The io stuff can be disabled, it should be split to different lib anyway.

If you can find repro for the crash, that'd be great.

EDIT: -DNON_POSIX flag should make it more windows friendly (no posix syscall stuff), but didn't test it on windows. Also updated to compile against latest v8.

**Comment by imzyxwvu** [comment]
En, I modified the code to make it able to run on Windows.
